doc/adoc/user_guide.adoc

```diff
@@ -273,7 +278,6 @@ a pointer to the respective log file.
   cloud framework is concerned, you will still be running an instance
   of the SLES version you started with. This cannot be changed.
 * The only supported migration path in the Public Cloud is from the
-  final 2 service packs of a distribution to the first service pack of
-  the next distribution. For example from SLES 12 SP4 or SLES 12 SP5 to
-  SLES 15 SP1. The packages delivered by SUSE in the Public Cloud Module
+  final 2 service packs of a distribution. For example from SLES 12 SP4 or SLES 12 SP5 to
+  SLES 15 SPX or . The packages delivered by SUSE in the Public Cloud Module
```

Comment:
`SLES 15 SPX or . ` or ?

Reply:
No SPX cannot be supported, well ultimately that is no longer the Public Cloud Teams decision. But ultimately if you want to support SPX, meaning any SP of the next distribution you have to build a migration iso based on every SPX due to the potential changes in zypper and the update stack. There are cases where zypper on 15 SP1 may not be able to migrated a target system to 15 SP3. The consistency of the update stack of the migration image and the migration target is important.

Reply:
If we are providing migration_target, it would be good to list any known supported migrations. If none other than SLES-15-SP1 exist, we need to clearly state these others are unsupported